plugins/kotlin/idea/tests/testData/parameterInfo/functionCall/NotGreen.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_FIR
```

Comment:
This was only coincidentally passing before. When there are multiple candidates, we don't get the argument mapping. We use it more now with these changes, so this test is now failing.